pydis_site/apps/content/resources/guides/python-guides/vps-services.md

```diff
@@ -0,0 +1,25 @@
+---
+title: VPS Services
+description: On different VPS services
+---
+If you need to run your bot 24/7 (with no downtime), you should consider using a virtual private server (VPS).
+This is a list of VPS services that are sufficient for running Discord bots.
+
+* https://www.scaleway.com/
+    * Based in Europe.
+* https://www.digitalocean.com/
+    * US-based.
+    * Considered by many to be the gold standard.
+    * Locations available across the world.
+* https://www.ovh.co.uk/
+    * France and Canadian locations available.
+* https://www.time4vps.eu/
+    * Seemingly based in Lithuania.
+* https://www.linode.com/
+    * Cheap VPS.
+* https://www.vultr.com/
+    * US-based, DigitalOcean-like.
+* https://galaxygate.net/
+    * A reliable, affordable, and trusted host.
```

Comment:
The URLs aren't hyperlinked. Let's get them to do that. Removing backslashes to keep the formatting consistent with other guides.

To change them:
```suggestion
* [https://www.scaleway.com](https://www.scaleway.com)
    * Based in Europe.
* [https://www.digitalocean.com](https://www.digitalocean.com)
    * US-based.
    * Considered by many to be the gold standard.
    * Locations available across the world.
* [https://www.ovh.co.uk](https://www.ovh.co.uk)
    * France and Canadian locations available.
* [https://www.time4vps.eu](https://www.time4vps.eu)
    * Seemingly based in Lithuania.
* [https://www.linode.com](https://www.linode.com)
    * Cheap VPS.
* [https://www.vultr.com](https://www.vultr.com)
    * US-based, DigitalOcean-like.
* [https://galaxygate.net](https://galaxygate.net)
    * A reliable, affordable, and trusted host.
```

Here's the current state:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/15021300/159114001-c148c9f0-2bc3-44c0-ba9d-f0208fe03dfd.png">